### PR TITLE
AIX Binary Support, Renamed binary files

### PR DESCRIPTION
**Pull request by ckim-tw**

### CODE REVIEW

File: bin/protoc [hunk 1]
```diff
@@ -16,7 +16,12 @@ def platform_arch
 end
 
 def exec_protoc_for_os(os)
-  exec("#{File.dirname(__FILE__)}/protoc-#{Protoc::PROTOBUF_VERSION}-#{os}-#{PLATFORM_TO_PROTOC_ARCH[platform_arch]}.exe", *ARGV)
+  case Platform::OS
+    when :win32
+      exec("#{File.dirname(__FILE__)}/protoc-#{Protoc::PROTOBUF_VERSION}-#{os}-#{PLATFORM_TO_PROTOC_ARCH[platform_arch]}.exe", *ARGV)
```
Comment (kraigstrong) [inline bin/protoc:21]: Is it better to define the filename once, and just append .exe if windows? That seems less error prone.

Comment (mbudnek) [reply bin/protoc:21]: Pretty sure avoiding this `case` was the reason the binaries all had '.exe' on the end in the first place.

**Comment by mbudnek** [comment]
Can you update the readme to note that the AIX binary isn't from maven central?

**Comment by mbudnek** [comment]
Also, since you needed to patch protobuf to get it to build on AIX, can you add that patch somewhere?

**Comment by ckim-tw** [comment]
> Can you update the readme to note that the AIX binary isn't from maven central?
> Also, since you needed to patch protobuf to get it to build on AIX, can you add that patch somewhere?

Yep.  I have it on a separate branch for now.  I will add comments about the patch too.  👍

**Pull request merged**